controlplane-rs/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM --platform=$BUILDPLATFORM tonistiigi/xx AS xx
+
```

Comment:
What is the `xx` image all about? I'm not sure about this :thinking:

Reply:
https://github.com/tonistiigi/xx
tldr: it contains helper scripts for common tools like `cargo`, `go`, etc. to make building cross platform docker images easy

Reply:
These kinds of dependencies always make me a bit wary. If you think it adds a lot of value then maybe we can move forward with it, but if its only keeping us from writing say, a couple lines? I would consider removing it from the supply-chain.

Reply:
I second Shane's concern. Looking through the repo, I can see the value-add---setting up cross-platform images can be a pain in the booty.

That said, I'm always inclined to err on the side of keeping your dependency footprint smaller, especially for security purposes.

Reply:
What are your thoughts on this one @aryan9600?

Reply:
Since this is sorta blocking all other work on the control-plane, I opted for a follow-up so we don't have to hold this PR up: https://github.com/kubernetes-sigs/blixt/issues/256